torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -707,12 +707,12 @@ def rewrite_node(node):
                 return g.call_function(ttnn.reshape, args=(args[0], output_size))
 
             if node.target in [torch.ops.aten.transpose.int, torch.ops.aten.t.default]:
+                rank = len(node.meta["val"].size())
+                if rank < 2:
+                    # Less 2D transpose is no-op
+                    return args[0]
                 if node.target == torch.ops.aten.t.default:
-                    rank = len(node.meta["val"].size())
                     assert rank >= 0 and rank <= 2, "Input tensor can only be 0D, 1D or 2D"
```

Comment:
this assert does not make sense anymore, right?

Reply:
It is still the assumption given by the torch.t op https://pytorch.org/docs/stable/generated/torch.t.html. I'm fine to remove it if you think it isn't necessary

Reply:
We can't remove, but given the above `if rank < 2:`, we might want to adjust

Reply:
I see. Changed the assert to `rank == 2` as it should be the only possible case when reaching this line